package.json

```diff
@@ -43,6 +43,114 @@
     "test": "vscode-test"
   },
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "ocaml-onboarding",
+        "title": "OCaml Onboarding",
+        "description": "Learn how to install OCaml, set up opam, and configure essential tools for development.",
+        "steps": [
+          {
+            "id": "install-opam-windows",
+            "when": "isWindows",
+            "title": "Install opam",
+            "description": "opam is the package manager for OCaml. Installing opam also installs the OCaml compiler. On Windows 10 or 11, install it using Winget:\n\nRun `winget install Git.Git OCaml.opam` in PowerShell.\n\n[Install opam](command:ocaml.install-opam)",
```

Comment:
Is running winget what will happen when pressing the button ? Let's make that explicit by calling the button: 'Open a terminal and run winget'

Same for the others.